cli/src/ios/update.ts

```diff
@@ -340,6 +341,11 @@ function filterARCFiles(plugin: Plugin) {
   return sourcesARC.length > 0;
 }
 
+function removeNoSystem(library: string, sourceFrameworks: Array<string>) {
```

Comment:
I think we should start preferring this syntax for TS arrays. Thoughts?

```suggestion
function removeNoSystem(library: string, sourceFrameworks: string[]) {
```

Reply:
I have no preference on one syntax over the other, but yeah, we should decide one or the other and always use the same.
Won't change it for now, but we can do a refactor later that replaces all of them.